content-src/components/ActivityFeed/ActivityFeed.scss

```diff
@@ -61,10 +61,7 @@
     height: auto;
     left: 0;
     margin-left: -4px;
-    margin-top: -($feed-gutter / 2 + 18 / 2);
-    position: absolute;
-    top: 50%;
-    width: auto;
+    margin-top: -($feed-gutter-v / 2 + 18 / 2);
```

Comment:
Why the extra math of `18 / 2` vs just `9`?
Or should we make `18` some sort of variable if it has significance?